TESTED.md

```diff
@@ -0,0 +1,11 @@
+#  Tested with S3
+
+### List of applications that have been tested with S3 Server
+
+- [Cyberduck](https://cyberduck.io/?l=en)
+- [Cloud Explorer](https://github.com/scality/S3/issues/277)
+- [AWS SDK](http://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html)
+- [s3curl](https://github.com/rtdp/s3curl)
+- [s3cmd](http://s3tools.org/s3cmd)
+- [fog (Ruby)](http://fog.io/storage/)
+- [boto (python)](https://github.com/boto/boto)
```

Comment:
We should separate them (boto2 and boto3) and provide sample config for each of them. Both have different options for setup.